raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -2285,7 +2287,11 @@ QuorumState quorum() {
     }
 
     public OptionalLong highWatermark() {
-        return quorum.highWatermark().isPresent() ? OptionalLong.of(quorum.highWatermark().get().offset) : OptionalLong.empty();
+        if (quorum.highWatermark().isPresent()) {
+            return OptionalLong.of(quorum.highWatermark().get().offset);
+        } else {
+            return OptionalLong.empty();
+        }
```

Comment:
No functional change here. Just a formatting change. Always found this line hard to read and I had to fix it :smile: